src/case-swapping.js

```diff
@@ -15,7 +16,17 @@
  * @return {string}
  */
 function swapCase(phrase) {
-  // write code here
+  let resultPhrase = '';
```

Comment:
what does result mean?
```suggestion
  let swappedPhrase = '';
```